src/components/Login.tsx

```diff
@@ -0,0 +1,18 @@
+import { LOGIN_URL, LOGIN_URL_LOCAL } from "../config/constants";
+
+const Login = () => {
```

Comment:
This is a Functional component; React seems to be moving towards using these a lot more. Same thing as using "class Login extends Component...", but for components that dont require state (ie are purely presentational), best practice seems to be using a functional component